src/commands.ts

```diff
@@ -72,19 +74,10 @@ export class Commands
 
     register()
     {
-        function register(receiver, name) {
-            let fn = receiver[name].bind(receiver);
-            return vscode.commands.registerCommand('rock.' + name, _ => fn());
-        }
-
-        for (const key of ['selectPackage',
-                           'buildPackage',
-                           'selectPackageType',
-                           'setDebuggingTarget',
-                           'debugPackage'])
-        {
-            this._context.extensionContext.subscriptions.
-                push(register(this, key));
-        }
+        this._vscode.registerAndSubscribeCommand('rock.selectPackage', this.selectPackage);
```

Comment:
This broke all commands. You are passing a reference to the method definition (`this` is undefined in such context).

Reply:
I noticed this morning ;-). As wrote in the message, I didn't test the extension yet (just passed the test suite)